Task4.py

```diff
@@ -0,0 +1,11 @@
+﻿""" 4. Дан список чисел. Посчитайте, сколько
+в нем пар элементов, равных друг другу. Считается,
+что любые два элемента, равные друг другу
+образуют одну пару, которую необходимо посчитать."""
+list_of_numbers = input().split()
+count = 0
+for i in range(len(list_of_numbers)):
+    for j in range(i + 1, len(list_of_numbers)):
```

Comment:
O(n**2) and it could be solved in O(n)

Reply:
The solution is simple, through dict. It was at a lecture. I did not immediately figure it out, so I did it differently.